boring/Cargo.toml

```diff
@@ -19,24 +19,27 @@ rustdoc-args = ["--cfg", "docsrs"]
 [features]
 # Controlling the build
 
-# Use a FIPS-validated version of BoringSSL. This feature sets "fips-compat".
-fips = ["fips-compat", "boring-sys/fips"]
-
-# Use a FIPS build of BoringSSL, but don't set "fips-compat".
+# NOTE: This feature is deprecated. It panders to the submoduled
+# boringssl-fips, which is extremely old and requires modifications to the
+# bindings, as some newer APIs don't exist and some function signatures have
+# changed. It is highly recommended to use `fips-precompiled` instead.
 #
```

Comment:
Since we haven't actually published 4.16, I would argue that we could just yank the tag and delete `fips-no-compat` before we cut and publish 4.17.

Reply:
I'm not sure what you mean, we didn't release 4.16 so we can just delete the code containing `fips-no-compat` and release 4.16